plugins/configuration.rb

```diff
@@ -0,0 +1,92 @@
+module Jekyll
+  class ConfigurationBlock < Liquid::Block
+    TYPE_LINKS = {
+      'action'       => '/docs/automation/action/',
+      'device_class' => '/components/binary_sensor/',
+      'template'     => '/docs/configuration/templating/',
+    }
+
+    REQUIRED_VALUES = {
+      true        => 'Required',
```

Comment:
These can never happen because by the time you test against this hash, you already assume it's a string.

Reply:
I changed the logic up to allow custom strings instead of using a hash.